### PR TITLE
Update the design of the wizard

### PR DESCRIPTION
**Pull request by Ayagoumi**

### CODE REVIEW

File: src/components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,176 @@
+<template>
```
Comment (aeddaqqa) [inline src/components/Footer.vue:1]: all the links in the footer redirect to chain4travel website

Comment (Ayagoumi) [reply src/components/Footer.vue:1]: You are right, will change it

File: src/components/Footer.vue [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    <div id="footer">
+        <div class="first-section">
+            <img src="@/assets/DarkModeLogo.svg" class="logo" />
```
Comment (aeddaqqa) [inline src/components/Footer.vue]: the logo needs to be adapted to the light mode 
Screenshot 2023-04-28 at 12 26 26

File: src/views/HomeNextStep.vue [hunk 3]
```diff
@@ -112,87 +82,73 @@ export default class Home extends Vue {
...
-        align-items: center;
-        .v-btn {
-            min-width: 175px;
+        &--desc {
+            font-size: 1.125rem;
+            line-height: 1.5;
         }
     }
-}
-.warningg {
```
Comment (aeddaqqa) [inline src/views/HomeNextStep.vue:159]: removing this class leads to have the attetion word without color, fix the color of the text for the "ATTENTION" word

File: src/views/HomeNextStep.vue [hunk 4]
```diff
@@ -74,36 +60,20 @@ export default class Home extends Vue {
 
 <style scoped lang="scss">
 @use "../styles/main";
-.text--bold {
```
Comment (aeddaqqa) [inline src/views/HomeNextStep.vue:77]: this text needs to be bold

**Review by aeddaqqa — DISMISSED** [review]

**Review by aeddaqqa — APPROVED** [review]

**Pull request merged**